src/Microsoft.AspNet.Mvc.Razor/TagHelpers/UrlResolutionTagHelper.cs

```diff
@@ -108,6 +107,11 @@ public override int Order
         /// <inheritdoc />
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
+            if (output == null)
```

Comment:
Will add `context` check in other tag helpers where I left it out. Will also update the `protected` methods in this class -- `protected` is the new `public`.